octobot_commons/pretty_printer.py

```diff
@@ -149,11 +149,13 @@ def _get_markdown_pretty_portfolio_row(
     :return: the portfolio row adapted for a markdown format
     """
     str_currency = "{:<4}".format(currency)
-    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=6))
+    str_holdings = "{:<12}".format(get_min_string_from_number(holdings, max_digits=4))
     str_ref_market_value = "{:<12}".format("")
     if ref_market:
         str_ref_market_value = "{:<12}".format(
-            get_min_string_from_number(ref_market_value) if ref_market_value else ""
+            get_min_string_from_number(ref_market_value, max_digits=2)
++            if ref_market_value
```

Comment:
I don't understand why you added a `+` here

Reply:
Omg, stupid typo, copy/paste error.. 🙈